kubernetes/apps/charts/sentry/Chart.yaml

```diff
@@ -6,5 +6,5 @@ version: 0.1.0
 appVersion: "1.4.2"
 dependencies:
   - name: sentry
-    version: 17.0.0
+    version: 17.10.1
```

Comment:
Is there any particular feature or fix we're bringing in with this minor version change, or just keeping up to date with non-breaking updates?